src/components/base.js

```diff
@@ -1,39 +1,64 @@
-import { PureComponent } from 'react';
 import { injectGlobal } from 'styled-components';
 import reset from 'styled-reset';
 
-export default class extends PureComponent {
-  componentDidMount() {
-    return injectGlobal`
-      ${reset}
-
-      @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500');
-
-      body, html {
-        font-size: 16px;
-        font-family: "Roboto", Helvetica, sans-serif;
-        -webkit-font-smoothing: antialiased;
-        -moz-osx-font-smoothing: grayscale;
-        line-height: 1.5;
-        height: 100%;
-      }
-
-      * {
-        box-sizing: border-box;
-      }
-
-      *, *:before, *:after {
-        box-sizing: border-box;
-      }
-
-      #___gatsby {
-        height: 100%;
-        display: block;
-      }
-    `;
-  }
-
-  render() {
-    return this.props.children;
-  }
-}
+const baseStyles = () => {
+  return injectGlobal`
+    ${reset}
+
+    @font-face {
+      font-display: fallback;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 400;
+      src: url(https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2) format('woff2');
```

Comment:
is there a risk that this url expires?

Reply:
No more than if you were to use `@import` - this is the same URL as in the CSS you get from Google. (see https://fonts.googleapis.com/css?family=Roboto)

I'm not sure exactly why but doing it this way seems to improve the FOUC a lot, and also allows us to control the `font-display` property.